frontend-plugin-core/src/test/java/com/github/eirslett/maven/plugins/frontend/lib/UtilsTest.java

```diff
@@ -0,0 +1,63 @@
+package com.github.eirslett.maven.plugins.frontend.lib;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class UtilsTest {
+
+  @Test
+  public void testImplode() {
+    final String separator = "Bar";
+    final List<String> elements = new ArrayList<String>();
+
+    assertEquals("", Utils.implode(separator, elements));
+
+    elements.add("foo");
+    elements.add("bar");
+    assertEquals("foo bar", Utils.implode(separator, elements));
+  }
+
+  @Test
+  public void testIsRelative() {
+    assertTrue(Utils.isRelative("foo/bar"));
+
+    assertFalse(Utils.isRelative("/foo/bar"));
+    assertFalse(Utils.isRelative("file:foo/bar"));
+    assertFalse(Utils.isRelative("C:\\SYSTEM"));
+  }
+
+  @Test
+  public void testMerge() {
+    final List<String> first = new ArrayList<String>();
+    first.add("foo");
+    final List<String> second = new ArrayList<String>();
+    second.add("bar");
+    final List<String> actual = Utils.merge(first, second);
+
+    final List<String> arrayList = new ArrayList<String>();
+    arrayList.add("foo");
+    arrayList.add("bar");
+
+    assertEquals(arrayList, actual);
```

Comment:
I guess it's technically correct, but maybe this would be easier to read:
```
assertEquals(
  Arrays.asList("foo", "bar"),
  Utils.merge(Arrays.asList("foo"), Arrays.asList("bar")
)
```
or what do you think?

Reply:
Yes, I agree. I am all in favour of increasing readability!
I have changed as suggested and similarly for `testPrepend()`.